src/articles/ArticlePreview.js

```diff
@@ -107,47 +111,64 @@ export default function ArticlePreview({
 
   return (
     <s.ArticlePreview>
-      <SmallSaveArticleButton
-        api={api}
-        article={article}
-        isArticleSaved={isArticleSaved}
-        setIsArticleSaved={setIsArticleSaved}
-      />
-
-      <s.Title>{titleLink(article)}</s.Title>
-      <ArticleSourceInfo
-        articleInfo={article}
-        dontShowPublishingTime={dontShowPublishingTime}
-        dontShowSourceIcon={dontShowSourceIcon}
-      ></ArticleSourceInfo>
-      <s.ArticleContent>
-        {article.img_url && <img alt="" src={article.img_url} />}
-        <s.Summary>{article.summary}...</s.Summary>
-      </s.ArticleContent>
-
-      <s.BottomContainer>
-        <s.Topics>
-          {topics.map((topic) => (
-            <span key={topic}>{topic}</span>
-          ))}
-        </s.Topics>
-        <ArticleStatInfo
-          cefr_level={cefr_level}
-          articleInfo={article}
-        ></ArticleStatInfo>
-      </s.BottomContainer>
-      {article.video ? (
-        <img
-          alt=""
-          style={{ float: "left", marginRight: "1em" }}
-          src={
-            "https://img.youtube.com/vi/" +
-            extractVideoIDFromURL(article.url) +
-            "/default.jpg"
-          }
+      {!isUnfinishedArticle && (
+        <SmallSaveArticleButton
+          api={api}
+          article={article}
+          isArticleSaved={isArticleSaved}
+          setIsArticleSaved={setIsArticleSaved}
         />
-      ) : (
-        ""
+      )}
+
+      {isUnfinishedArticle && (
+        <div>
+          <s.UnfinishedArticleContainer>
+            <s.Title>{titleLink(article)}</s.Title>
+            {article.img_url && <img alt="" src={article.img_url} />}
+          </s.UnfinishedArticleContainer>
+          <div style={{ fontWeight: "550" }}>
```

Comment:
i'm not sure: isn't it nicer to separate the styling into CSS files? does it get harder to read in that way you think? Or is there a good reason to keep these inlines sometimes?

Reply:
I think to me in-lines make sense when it's a very small change, like this one, where I am just making the font a little different than <b>, but creating a styled component might also be a bit of "overkill" to have a "slight-less-bolder-text" - but maybe using a class would be nice, though I am not so sure how to just define a class in the Styled Component file without nesting it in another class. I will look into it, but for now I made styled components for everything.